apps/www/pages/security.mdx

```diff
@@ -126,8 +126,7 @@ In addition to internal security reviews, we use various tools to scan our code
 
 <h3 className="text-3xl">Security Portal</h3>
 
-Access Supabase’s security and compliance documents on our Security Portal, such as
-penetration testing and audit reports.
+Access more of Supabase’s security on our Security Portal, such as penetration testing and audit reports.
```

Comment:
Maybe "Access more of Supabase’s security **information** on our Security Portal"? Or "Access more **about** Supabase’s security"?

Reply:
ah sorry - merged it.

make another PR if you want :D

Reply:
i'll make this change in my PR!